front-end/src/helpers/resizeInput.js

```diff
@@ -1,22 +1,25 @@
-const resizeInput = (el) => {
+const resizeInput = (arr) => {
```

Comment:
I'm thinking resizeInputs (with 's') would be a better name, that would suggest it accepts an array.